SkillChat.Server.ServiceModel/Molds/MessageMold.cs

```diff
@@ -26,6 +26,12 @@ public class MessageMold
         [Description("Идентификатор чата")]
         public string ChatId { get; set; }
 
+        [Description("Идентификатор редактирования сообщения")]
+        public bool Edited { get; set; }
+
+        [Description("Время отредактированной публикациия")]
+        public DateTimeOffset EditedPostTime { get; set; }
```

Comment:
Я бы предложил более логичное название
```suggestion
        public DateTimeOffset LastEditTime { get; set; }
```